docker/testnet/stacks/Config.toml

```diff
@@ -0,0 +1,89 @@
+[node]
+working_dir = "/root/stacks-blockchain/data"
+rpc_bind = "0.0.0.0:20443"
+p2p_bind = "0.0.0.0:20444"
+bootstrap_node = "029266faff4c8e0ca4f934f34996a96af481df94a89b0c9bd515f3536a95682ddc@seed.testnet.hiro.so:30444"
+always_use_affirmation_maps = true
+require_affirmed_anchor_blocks = true
+pox_sync_sample_secs = 30
+prometheus_bind = "0.0.0.0:9153"
+# If you have an existing node with this value set to true, keep it as true.
+stacker = false
+
+[burnchain]
+mode = "krypton"
+peer_host = "bitcoin"
+peer_port = 18444
+pox_prepare_length = 100
+pox_reward_length = 900
+
+[connection_options]
+auth_token = "XXX"
+private_neighbors = false
+
+# Add sbtc-signer as an event observer.
+# NOTE: See the readme for how to set this value correctly, your endpoint
+# may not be named `sbtc-signer:8801`.
+#
+# It's perfectly fine to have multiple event observers, just add this
+# section below your existing one, it'll adds the event observer to the
+# events_observers list.
```

Comment:
Should we add a note here that while it's fine, you one of them is down/not responding it will block the stacks node? (unless they already fixed this in stacks-core)

Reply:
They didn't, adding a note